cypress/e2e/tests/administration/repository/maven.test.ts

```diff
@@ -144,8 +139,8 @@ describe(["@tier1"], "Test secure and insecure maven repository analysis", () =>
         });
     });
 
-    afterEach("Reset Url", function () {
-        resetURL();
+    afterEach("Clear state", function () {
+        Analysis.open(true);
```

Comment:
This can be reverted , it is doing the same thing as resetURL()

Reply:
you're right it passed after revert, https://main-jenkins-csb-migrationqe.apps.ocp-c1.prod.psi.redhat.com/job/mta/job/mta-ui-tests-runner/3323/console